config/routes.rb

```diff
@@ -131,6 +131,14 @@
     end
   end
 
+  if defined?(Sufia::StatsAdmin)
+    namespace :admin do
+      constraints Sufia::StatsAdmin do
```

Comment:
@jcoyne That is still at the controller level, so the route exists but they are not authorized.  It is a slightly different pattern than we are currently using for administrative routes.  I do not mind changing the pattern overall, but that is not within scope of this PR.  I can add a ticket that can be addressed later.